src/component/parser/json/converter/CategoryConverter.ts

```diff
@@ -26,10 +26,8 @@ export default class CategoryConverter {
   constructor(private convertedElements: ConvertedElements) {}
 
   deserialize(definitions: TDefinitions): void {
-    ensureIsArray<TCategory>(definitions.category).forEach(category => {
-      ensureIsArray(category.categoryValue).forEach(categoryValue => {
-        this.convertedElements.registerCategoryValues(categoryValue.id, categoryValue.value);
-      });
-    });
+    ensureIsArray<TCategory>(definitions.category).forEach(category =>
+      ensureIsArray(category.categoryValue).forEach(categoryValue => this.convertedElements.registerCategoryValues(categoryValue.id, categoryValue.value)),
+    );
```

Comment:
❓ for this "double" forEach call, I am not very happy with the result when removing the whole braces.
We could remove them only in the inner for each.

```typescript
ensureIsArray<TCategory>(definitions.category).forEach(category => {
      ensureIsArray(category.categoryValue).forEach(categoryValue => this.convertedElements.registerCategoryValues(categoryValue.id, categoryValue.value));
});
```

Reply:
For me, it's good without the whole braces 🙂

Reply:
Ok so let's keep the change 😸